shell/src/test/java/tachyon/shell/TfsShellTest.java

```diff
@@ -1072,4 +1072,55 @@ public void unsetTTLTest() throws Exception {
     Assert.assertEquals(0, mFsShell.run("unsetTTL", filePath));
     Assert.assertEquals(Constants.NO_TTL, mTfs.getInfo(file).getTtl());
   }
+
+  @Test
+   public void setIsPinnedTest() throws Exception {
+    String filePath = "/testFile";
+    TachyonFile file = TachyonFSTestUtils.createByteFile(mTfs, filePath, TachyonStorageType.STORE,
+        UnderStorageType.NO_PERSIST, 1);
+
+     //Ensure that the file exists first
+    Assert.assertTrue(fileExist(new TachyonURI(filePath)));
+
+     //By default the file in pinned , so must be unpinned first
+    Assert.assertEquals(false, mTfs.getInfo(file).isPinned);
```

Comment:
I think you can use assertFalse() and assertTrue() when you want to compare with `false` and `true` throughout this file.